ThunderBasics/DesignableView.swift

```diff
@@ -77,11 +77,12 @@ public class ScaleBasedConstraint: NSLayoutConstraint {
     open override func prepareForInterfaceBuilder() {
         
         super.prepareForInterfaceBuilder()
+        // We don't set layer.shadowColor here because it clashes when compiling
+        // with `UILabel`'s own `shadowColor` property.
```

Comment:
Should we have a `shadowLayer` on the `UILabel`. Apple put it on the layer, but then against they've done `backgroundColor` on both etc?

Reply:
Possibly, to be honest it's not very often we put a shadow on a `UILabel` itself, without it being contained in another subview. May have to revisit if we do see issues!